.markdownlint.yaml

```diff
@@ -0,0 +1,3 @@
+"default": true # Default state for all rules
+"MD013": false # Disable rule for line length
```

Comment:
[Non Blocking][Suggestion]
In celestia-app, we disabled the following rule:
https://github.com/celestiaorg/celestia-app/blob/45353941dc365630b42b096fb587044ab6fa36f2/.markdownlint.yaml#L2-L3
Does it make sense to also disable it here? since all repos will inherit the rules from this repo, apparently.

Reply:
no, each repo has it's own `markdownlint.yaml` file. Refactoring the action simply pulls the execution logic out, but allows each repo to maintain their own linting rules. 👍

Reply:
cool